package.json

```diff
@@ -19,18 +19,19 @@
     "homepage": "https://github.com/alexemanuelol/rustPlusPlus#readme",
     "dependencies": {
         "@discordjs/rest": "^1.0.1",
-        "discord.js": "^14.2.0",
-        "discord-api-types": "^0.37.1",
-        "colors": "^1.4.0",
-        "rustplus.js": "github:liamcottle/rustplus.js",
-        "winston": "^3.3.3",
-        "push-receiver": "^2.1.1",
-        "lodash": "^4.0.0",
         "axios": "^0.26.1",
+        "colors": "^1.4.0",
+        "discord-api-types": "^0.37.1",
+        "discord.js": "^14.2.0",
+        "fuse.js": "^6.6.2",
         "jimp": "^0.16.1",
-        "translate": "^1.4.1"
+        "lodash": "^4.0.0",
+        "push-receiver": "^2.1.1",
+        "rustplus.js": "github:liamcottle/rustplus.js",
+        "translate": "^1.4.1",
+        "winston": "^3.3.3"
```

Comment:
Just a note this order is managed by npm.

When you do `npm install ...` it's going to organise the list.